### PR TITLE
[PLG21] adds Love Your LGS 2021 Promos

### PR DESCRIPTION
**Pull request by chesse20**
Implemented by request in the "Reddit Xmage" Discord

### CODE REVIEW

**Comment by xenohedron** [comment]
Verify failure:

`(missing NON_FULL_USE_VARIOUS, etc): PLG21 - Love Your LGS 2021 - Orb of Dragonkind - J1`

Within an `ExpansionSet`, if a single card has multiple printings (e.g. alt art or frame), then each printing needs to be tagged as `NON_FULL_USE_VARIOUS`. (not the clearest naming, a relic from times when multiple arts were infrequent). Example from DMU:

```
cards.add(new SetCardInfo("Academy Loremaster", 391, Rarity.RARE, mage.cards.a.AcademyLoremaster.class, NON_FULL_USE_VARIOUS));
cards.add(new SetCardInfo("Academy Loremaster", 40, Rarity.RARE, mage.cards.a.AcademyLoremaster.class, NON_FULL_USE_VARIOUS));
```

**Comment by chesse20** [comment]
got it

File: Mage.Sets/src/mage/sets/LoveYourLGS2021.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+public class LoveYourLGS2021 extends ExpansionSet {
+
+    private static final LoveYourLGS2021 instance = new LoveYourLGS2021();
+
+    public static LoveYourLGS2021 getInstance() {
+        return instance;
+    }
+
+    private LoveYourLGS2021() {
+        super("Love Your LGS 2021", "PLG21", ExpansionSet.buildDate(2021, 1, 1), SetType.PROMOTIONAL);
```
Comment (xenohedron) [inline Mage.Sets/src/mage/sets/LoveYourLGS2021.java:19]: Scryfall seems to think the date is 2021, 6, 22

Comment (chesse20) [reply Mage.Sets/src/mage/sets/LoveYourLGS2021.java]: thanks

**Review by xenohedron — APPROVED** [review]

**Pull request merged**